index.js

```diff
@@ -78,13 +78,51 @@ class BookListApp {
     document.querySelector('.title').value = '';
     document.querySelector('.author').value = '';
   }
+  getDate(){
+    const time = document.querySelector('.time');
+    var date = new Date();
+    var now = date.toString().split(' ').splice(1,4);
+    let pe;
+    if(date.getHours() >= 12){
+      pe = 'pm'
+    }else{
+      pe = "am";
+    }
+    var mon = `${now[0]} ${now[1]}th ${now[2]}, ${now[3]} ${pe}`
+    time.innerHTML = mon;
+     
+  }
+  navLinks(){
+    // Switching Through Sections
+      const allSections = document.querySelectorAll(".sections")
+      const listCont = document.querySelector(".nav-items")
+      listCont.addEventListener("click", (e) =>{
+        e.preventDefault()
+        const id = e.target.dataset.id
+        if(id){
+
+          //Removing the active section
+          allSections.forEach((section)=>{
+          section.classList.remove("activesec")
+          })
+          //  Adding the active section
+          const targetedSec = document.getElementById(id)
+          targetedSec.classList.add("activesec")
+      }
```

Comment:
- [ ] **Required:** The switch isn't working correctly, kindly look into the code.

![switch](https://user-images.githubusercontent.com/27709832/193110903-30520ff7-a2b4-47ef-aa54-cc06aa7a2009.gif)

### What can you do:
- Ensure that the container containing each section is displayed when the nav is active and all others are hidden.